CHANGELOG.rst

```diff
@@ -1,6 +1,8 @@
 Change Log
 ==========
 
+* :bug: Adding a new `Activity2` to a parent now updates the `_cached_children`.
```

Comment:
```suggestion
* :bug: Adding a new `Activity2` to a parent now cache its childeren in case the parent also have the children cached. In essence it updates the `_cached_children`. #722 
```